monitoring/src/main/scala/akkaviz/server/WebSocketRepl.scala

```diff
@@ -0,0 +1,178 @@
+package akkaviz.server
+
+import java.io.{InputStream, OutputStream, PrintStream}
+import java.util.concurrent.Executors
+
+import akka.http.scaladsl.model.ws._
+import akka.http.scaladsl.server.{Directives, Route}
+import akka.stream.scaladsl._
+import akka.util.ByteString
+import ammonite.ops.Path
+import ammonite.repl._
+
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2101331339/issues/source?bid=3121282&fileBranchId=3142893#l13)